app/code/Magento/ProductAlert/view/frontend/templates/product/view.phtml

```diff
@@ -4,10 +4,15 @@
  * See COPYING.txt for license details.
  */
 ?>
-<?php /* @var $block \Magento\ProductAlert\Block\Product\View */?>
+<?php
```

Comment:
Please remove not needed opening and closing PHP tags

Reply:
fixed